src/HashSetHashMap/HashMap/TwoSum/TwoSum.java

```diff
@@ -0,0 +1,41 @@
+package HashSetHashMap.HashMap.TwoSum;
+
+import org.junit.Test;
+
+import java.util.PriorityQueue;
+import java.util.Queue;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+
+/**
+ * Ransom Note | Leetcode #0001 | Easy

```

Comment:
The title is incorrect, and this is repeated in the Arrays section. It falls into both, of course, but I don't think we should repeat the question.